core/iwasm/interpreter/wasm_interp_classic.c

```diff
@@ -70,6 +70,31 @@ typedef float64 CellType_F64;
             goto unaligned_atomic;                                   \
     } while (0)
 
+#if WASM_ENABLE_DEBUG_INTERP != 0
+#define TRIGGER_WATCHPOINT_SIGTRAP()                          \
```

Comment:
Please use `do { .. } while (0)` to encapsulate the code, or when we call the macro `TRIGGER_WATCHPOINT_SIGTRAP();`, there will be two `;` when the macro is expanded, which might cause compile warning.